observability/otel/otel-collector/otel-collector-exporters.adoc

```diff
@@ -189,6 +189,46 @@ include::snippets/technology-preview.adoc[]
 <8> Defines how long metrics are exposed without updates. The default is `5m`.
 <9> Adds the metrics types and units suffixes. Must be disabled if the monitor tab in Jaeger console is enabled. The default is `true`.
 
+[id="prometheus-remote-write-exporter_{context}"]
+== Prometheus Remote Write Exporter
+
+The Prometheus Remote Write Exporter exports metrics to compatible back ends.
+
+[WARNING]
```

Comment:
Might be hard here, but try to avoid stacked admonitions. See https://redhat-documentation.github.io/supplementary-style-guide/#admonitions

Reply:
Yes, both admonitions are critical here.